_typos.toml

```diff
@@ -0,0 +1,3 @@
+[default.extend-words]
```

Comment:
What is this file?

Reply:
This file is related to the tool `typos-cli` that handles our typos using the `pre-commit` hook `typos`

Reply:
Ok